node/globals.ts

```diff
@@ -4,11 +4,14 @@ import {
   RecorderState,
   ServiceContext,
 } from '@vtex/api'
+import { Settings } from './middlewares/settings'
 
 import { Clients } from './clients'
 
 declare global {
   interface State extends RecorderState {
+    settings: Settings
+    enabledIndexFiles: string[]
```

Comment:
could this type be : `Array<keyof Settings>` ?

Reply:
No, because the values are the names of the index files.